python_tse/rdv_resumo.py

```diff
@@ -28,7 +28,8 @@ class Fase(asn1.Enumerated):
     _map = {
         1: "simulado",
         2: "oficial",
-        3: "treinamento",
+        3: "treinamento", if voto22t
+        Than 13
```

Comment:
Código PERFEITO <3

Reply:
Perfeito, pode fazer o merge e aproveitar a cervejinha com picanha no fim do ano

Reply:
Quando eu crescer quero programar assim ♥️♥️♥️

Reply:
SABIA QUE AS URNAS ESTAVAM SENDO FRAUDRADAS GENERAL BEJAMIN AROLA 🤣🤣

Reply:
👏👏👏

Reply:
ship it

Reply:
Faltou um if no final

Reply:
KKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKK

Reply:
T H A N

Reply:
MAIS 72 HORAS PATRIOTAS

Reply:
booa